### PR TITLE
Use ordered process list for CSV upload instructions

### LINKED ISSUE #4254: Update "Submit Spreadsheet Results" Page
Users are bypassing the guide and the necessary steps to properly format their CSVs prior to submitting (based on research completed during the onboarding sessions).  We should update the steps and call to action.  

[Figma](https://www.figma.com/file/NPSPM5SBzQ7Ra3mmr9M9cB/SimpleReport-Public-Site?node-id=13362%3A35098)

Acceptance criteria

- [ ] Update the copy and order of the steps per the Figma file
- [ ] Change the styling to match Figma (links should be blue)

### PR DESCRIPTION
**Pull request by nathancrtr**
## Related Issue

- Closes #4254 

## Changes Proposed
- Use an ordered process list (from USWDS) for CSV upload instructions

## Screenshots / Demos
Screen Shot 2022-10-28 at 12 17 57 PM

Compare with design in [Figma](https://www.figma.com/proto/NPSPM5SBzQ7Ra3mmr9M9cB/SimpleReport-Public-Site?node-id=13362%3A35098&scaling=min-zoom&page-id=13162%3A34468&starting-point-node-id=13362%3A34954).

### CODE REVIEW

**Review by DanielSass — APPROVED** [review]

File: frontend/src/app/testResults/uploads/Uploads.tsx [hunk 1]
```diff
@@ -179,9 +179,9 @@ const Uploads = () => {
         </div>
         <div className="usa-alert usa-alert--info margin-left-105em margin-right-105em maxw-tablet-lg">
           <div className="usa-alert__body">
-            <h3 className="usa-alert__heading">
+            <h2 className="usa-alert__heading">
               What is the bulk results uploader?
-            </h3>
+            </h2>
```
Comment (zdeveloper) [inline frontend/src/app/testResults/uploads/Uploads.tsx:184]: I am assuming the axe tools are ok with this change.

File: frontend/src/app/testResults/uploads/Uploads.tsx [hunk 2]
```diff
@@ -208,32 +208,50 @@ const Uploads = () => {
...
+              </li>
+              <li className="usa-process-list__item margin-bottom-1em">
+                <h2 className="usa-process-list__heading">
+                  Following the guide and template, format your data to match
+                  SimpleReport requirements
+                </h2>
+              </li>
+              <li className="usa-process-list__item margin-bottom-1em">
+                <h2 className="usa-process-list__heading">
+                  Save your spreadsheet as a CSV
```
Comment (zdeveloper) [inline frontend/src/app/testResults/uploads/Uploads.tsx:244]: @lauraykerr should this read as `CSV file` instead of `CSV`?

**Review by zdeveloper — DISMISSED** [review]

File: frontend/src/app/testResults/uploads/Uploads.tsx (hunk 2 above)
Comment (lauraykerr) [reply frontend/src/app/testResults/uploads/Uploads.tsx:244]: Thanks for pointing this out @zdeveloper — can we actually say `Save your spreadsheet in a CSV format` to be consistent with steps in the guide?

Comment (nathancrtr) [reply frontend/src/app/testResults/uploads/Uploads.tsx]: @zdeveloper @lauraykerr aye aye 🏴‍☠️ 

changed in [2837d12](https://github.com/CDCgov/prime-simplereport/pull/4619/commits/2837d124089d524e5b817c7dc967601dce7c2cb0).

**Review by mehansen — APPROVED** [review]
🚀

**Review by zdeveloper — APPROVED** [review]
Thank you!

**Pull request merged**